Source/Meadow.Foundation.Peripherals/Displays.TftSpi/Samples/ST7789_Sample/MeadowApp.cs

```diff
@@ -40,9 +44,9 @@ public MeadowApp()
             display = new St7789(
                 device: Device,
                 spiBus: spiBus,
-                chipSelectPin: Device.Pins.D15,
-                dcPin: Device.Pins.D11,
-                resetPin: Device.Pins.D14,
+                chipSelectPin: Device.Pins.A03,
+                dcPin: Device.Pins.A04,
+                resetPin: Device.Pins.A05,
```

Comment:
I don't remember touching this. Is this an intentional change or a potential regression?

Reply:
@profexorgeek you can use those pins as Digital IO ports. In this case, these pins are wired to the Project Lab board, thats why they're like this in this sample.